docs/src/main/asciidoc/azure-support-overview-concept.adoc

```diff
@@ -0,0 +1,117 @@
+////
+This document is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="azure-support-overview-concept"]
+= Quarkus on Microsoft Azure cloud platform
+include::_attributes.adoc[]
+:categories: cloud
+
+Quarkus provides support and capabilities to help you to build cloud-native applications that run on Microsoft Azure. Quarkus also provides support for building serverless Java functions by using Microsoft Azure Functions.
+You can deploy the container image of a Quarkus application that you have developed locally to the Microsoft Azure cloud platform.
+You can also entirely build, host, and run a Quarkus application on Azure by using some of the key developer services for Azure.
+
+== Microsoft Azure
+
+link:https://learn.microsoft.com/en-us/azure/?product=popular[Microsoft Azure] is a versatile and flexible cloud platform, which you can use to host your Quarkus applications.
+Azure supports various programming languages, complex architectures and frameworks, including Java and Quarkus.
+Azure also fully supports the Apache Maven Java development tool.
```

Comment:
Not sure what this refers to. 
Makes it sounds like you can use it with gradle? I assume that is not that case.

Reply:
Fixed. Removed the 'marketing spiel' and restructured the new integration overview concept topic.